.github/workflows/stable_diffusion_1_5_cpp.yml

```
@@ -0,0 +1,20 @@
+name: stable_diffusion_1_5_cpp
+on:
+  pull_request:
+    paths:
+      - image_generation/stable_diffusion_1_5/cpp/src/**
+      - .github/workflows/stable_diffusion_1_5_cpp.yml
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  stable_diffusion_1_5_cpp:
+    runs-on: ubuntu-20.04-8-cores
```

```suggestion
    runs-on: ubuntu-20.04
```
Can you try it? `causal_lm_cpp.yml` uses `ubuntu-20.04-8-cores` because open lama 3b is too big.